dts/arm/st/stm32f411.dtsi

```diff
@@ -5,3 +5,27 @@
  */
 
 #include <st/stm32f401.dtsi>
+
+/ {
+	soc {
+		spi4: spi@40013400 {
```

Comment:
SPI4 is present in stm32f401, move to stm32f401.dtsi

Reply:
left it as is, stm32f401 Socs mostly don't have spi4